### PR TITLE
[frontend] English grammar errors in the toggle labels for the "Update a group" panel (#8445)

### PR DESCRIPTION
**Pull request by SarahBocognano**
### Proposed changes

* See issue

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/8445

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/settings/groups/GroupEditionOverview.tsx [hunk 1]
```diff
@@ -209,7 +209,7 @@ const GroupEditionOverviewComponent: FunctionComponent<GroupEditionOverviewCompo
               type="checkbox"
               name="restrict_delete"
               label={t_i18n(
-                'Deny users to delete content not created by them',
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/private/components/settings/groups/GroupEditionOverview.tsx:212]: you might want to remove the corresponding intl keys ?

Comment (SarahBocognano) [reply opencti-platform/opencti-front/src/private/components/settings/groups/GroupEditionOverview.tsx:212]: It wasn't existing in the first place :( 
that's why I added translation for the other sentences too

**Comment by seanthegeek** [comment]
What about the marking definitions text?

**Review by labo-flg — APPROVED** [review]

**Comment by SarahBocognano** [comment]
Hi @seanthegeek , as my responsable of product said in the issue's comment part, we've decided to only change this text and not the one related to markings :)

**Pull request merged**